lib/utils.ts

```diff
@@ -59,15 +62,27 @@ export function expandTabs(line: string): string {
     });
 }
 
+function getRegexForTempdir(): RegExp {
+    const tmp = os.tmpdir();
+    return new RegExp(tmp.replaceAll('/', '\\/') + '\\/' + ce_temp_prefix + '[\\w\\d-.]*\\/');
```

Comment:
Is this trying to escape the `tmp` dir to be a regular exprssion-safe thing? I'm not sure it's all that robust (though it might work in all reasonable circumstances)